common/src/main/java/com/wynntils/utils/wynn/ItemUtils.java

```diff
@@ -65,4 +69,13 @@ public static MutableComponent getNonGearDescription(ItemStack itemStack, String
         }
         return null;
     }
+
+    public static boolean canBeWielded(ItemStack itemStack) {
```

Comment:
I am pretty certain we already do a similar check elsewhere, like in Quick Cast. Please reuse that code instead.

Reply:
the checks there are done in place. I can make quick cast use this method instead. but then I'd have to make it 2 methods because quick cast displays different messages depending on which condition you are not fulfilling.